src/Content/ContentStorageHandler.php

```diff
@@ -75,6 +75,20 @@ public function move(
 	 */
 	public function read(string $version, string $lang): array;
 
+	/**
+	 * Searches and replaces one or multiple strings in all versions
+	 *
+	 * @param string $lang Code `'default'` in a single-lang installation
+	 * @param array $map search-replace mapping where all keys are replaced by their values
+	 *
+	 * @throws \Kirby\Exception\NotFoundException If the version does not exist
+	 */
+	public function replace(
```

Comment:
In the context of the content storage handler classes, I would think that "replace" means that the content is replaced/overwritten. We use Version::replace like that. Maybe we could use something like replaceStrings?